.circleci/config.yml

```
@@ -24,63 +24,70 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - dependency-cache-{{ checksum "package-lock.json" }}
+            - dependency-cache-{{ checksum "yarn.lock" }}
       - run:
           name: Install Dependencies
-          command: npm i --force #Remove force once webex version is upgraded in sdk-component-adapter 
+          command: yarn install
       - save_cache:
-          key: dependency-cache-{{ checksum "package-lock.json" }}
-          paths: node_modules
+          key: dependency-cache-{{ checksum "yarn.lock" }}
+          paths:
+            - node_modules
+            - packages/*/node_modules
       - persist_to_workspace:
           root: .
-          paths: node_modules
+          paths:
+            - node_modules
+            - packages/*/node_modules
 
   linter:
     executor: node-executor
     steps:
       - checkout_from_workspace
       - run:
           name: Linting
-          command: npm run test:eslint
+          command: yarn workspace @webex/widgets run test:eslint
```

Do you think we should update all these commands to run the commands for each package? Rather than a specific package.

```suggestion
          command: "yarn workspaces foreach -A run test:eslint
```

---

I wanted to keep that in the feature branch since we are merging this to the main branch

---

Sure.